app/styles/layout/_header.scss

```
@@ -0,0 +1,361 @@
+.header {
+  font-family: $font-primary;
+
+  &__top {
+    display: none;
+
+    @media (min-width: $lg) {
+      & {
+        display: block;
+        background: $color-black-1;
+        font-size: $font-size-12;
+        color: $color-grey-dark-3;
+
+        &__left {
+          padding: 7px 0px;
+          & i {
+            margin: 0px 6px 0px 4px;
+          }
+        }
+
+        &__middle {
+          margin-left: 15px;
+          display: flex;
+          align-items: center;
+          height: 100%;
+
+          & a {
```

```suggestion
         a {
```
Không cần & nó cũng ăn mà anh

---

Đã fix